pandas/tests/extension/base/methods.py

```diff
@@ -21,10 +21,10 @@ class BaseMethodsTests(BaseExtensionTests):
     def test_hash_pandas_object(self, data):
         # _hash_pandas_object should return a uint64 ndarray of the same length
         # as the data
+        from pandas.core.util.hashing import _default_hash_key
```

Comment:
IIUC the issue is that hashing hasnt been imported into core.util.  Should we try to address that directly?

Reply:
I don't really care either way; it's not a public import, and internally we always import it correctly instead of using it from a top-level pandas object.

Reply:
> I don't really care either way

Agreed.  I'm fine with this.